src/Castle.Core.Tests/log4netIntegration/Log4netFactoryTestCase.cs

```diff
@@ -31,6 +29,7 @@ namespace Castle.log4netIntegration
 	[TestFixture]
 	public class Log4netFactoryTestCase
 	{
+		private const string log4netXmlPath = "./log4netIntegration/log4net.xml";
```

Comment:
Make sure you note in the changelog upgrading log4net to 2.0.8 and support for .NET Core, 2.0.7 didn't support .NET Core.